src/coreclr/vm/eepolicy.cpp

```diff
@@ -348,6 +348,25 @@ void LogInfoForFatalError(UINT exitCode, LPCWSTR pszMessage, LPCWSTR errorSource
 {
     WRAPPER_NO_CONTRACT;
 
+    static volatile LONG s_crashingThreadId = 0;
+
+    LONG currentThread = GetCurrentThreadId();
+    LONG previousThread = ::InterlockedCompareExchange(&s_crashingThreadId, currentThread, 0);
+
+    if (previousThread == currentThread)
+    {
+        PrintToStdErrA("Repeated fatal crash ignored to avoid infinite recursion.\n");
```

Comment:
Can you please change the message based on one of my suggestion?

Reply:
Fixed in 4th commit.